### DISCUSSION TITLE
Upgrade from 2.30 to 3.0.6: Cannot find module 'date-fns/format/index.js' from 'node_modules/date-fns-tz/format/index.js'

### DISCUSSION THREAD

**Discussion by ningji** [js-fns/date-fns#3642] [Q&A] [answered]
was on
date-fns:2.30.0
date-fns-tz:2.0.0

When upgraded to date-fns: 3.0.6 from 2.30.0 (date-fns-tz stays on v2.0.0 as this is their latest), i got this:

```
    Cannot find module 'date-fns/format/index.js' from 'node_modules/date-fns-tz/format/index.js'

    Require stack:
      node_modules/date-fns-tz/format/index.js
      node_modules/date-fns-tz/index.js
```

Any suggestions other than stay on the old version pls ?

**Comment by kossnocorp** [comment, marked as answer]
Hey, follow the progress here: https://github.com/marnusw/date-fns-tz/issues/260

There's nothing I can do from date-fns side
